specification/e-referrals-service-api.yaml

```diff
@@ -1397,10 +1397,12 @@ paths:
         ## Pre-requisites
 
         In order to use this endpoint you must be an authenticated e-RS user and use one of the following e-RS roles:
+          - `REFERRING_CLINICIAN`
+          - `REFERRING_CLINICIAN_ADMIN`
           - `SERVICE_PROVIDER_CLINICIAN`
 
         You need to use the `Unique Booking Reference Number` (UBRN) of the advice and guidance request. 
-        You may have identified the UBRN via [[HYPERLINK_A023]].
+        For service provider users, you may have identified the UBRN via [[HYPERLINK_A023]].
```

Comment:
Has this wording change been agreed with the program? How would referrers be able to identify UBRNs?